spec/form_models/publishers/vacancies/job_applications/change_email_address_form_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module Publishers
+  module Vacancies
+    module JobApplications
+      RSpec.describe ChangeEmailAddressForm do
+        context "with invalid email" do
```

Comment:
:question: if we're testing the validations in these forms. Shouldn't we also cover that when the data is present/valid the form is valid?